staging/src/k8s.io/apiserver/pkg/server/healthz.go

```diff
@@ -101,9 +101,14 @@ func (s *GenericAPIServer) installReadyz() {
 	s.readyzLock.Lock()
 	defer s.readyzLock.Unlock()
 	s.readyzChecksInstalled = true
-	healthz.InstallReadyzHandlerWithHealthyFunc(s.Handler.NonGoRestfulMux, func() {
-		// note: InstallReadyzHandlerWithHealthyFunc guarantees that this is called only once
+	healthz.InstallReadyzHandlerWithCustomFunc(s.Handler.NonGoRestfulMux, func() {
+		// note: InstallReadyzHandlerWithCustomFunc guarantees that this is called only once
 		close(s.hasBeenReadyCh)
+	}, func(_ int) bool {
```

Comment:
is there a readyz handler ever where we want stacktraces?

Reply:
do you want to default it to `false` ?

Reply:
yes, without the parameter

Reply:
ok, I'll update the PR. Thanks.